site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Community Plugins in the Marketplace
+subsection: Plugins (Beta)
+weight: 60
+---
+
+Once your plugin has reached a certain level of quality, you might consider submitting it to the Plugin Marketplace. The Plugin Marketplace allows discovery, installation and updates of plugins directly within Mattermost. It is a great way to get feedback on your plugin and help make it more popular. Once your plugin is accepted to the Marketplace, Mattermost will also send you swag!
+
+### Requirements for adding community plugin to the Marketplace
+Every community plugin must fulfill the following checklist to be added to the Marketplace:
+
+**Product Requirements (Checked by a Product Manager)**
+
+1. The plugin is published under an Apache v2 compatible license (e.g. no GPL, APGL). A list of compatible licenses can be found [here](https://apache.org/legal/resolved.html#category-a).
+2. The source code is available in a public git repository.
+3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
+4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features and a development guide. This is typically a README file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [README of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
+5. For every release a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
```

Comment:
It would be cleaner to say "the plugin manifest" here.

Separately, how should a developer handle the scenario where there are a number of previous releases that do not have changelog entries?

Reply:
Good catch. There only has to be a changelog for new releases. Made this more clear :+1: